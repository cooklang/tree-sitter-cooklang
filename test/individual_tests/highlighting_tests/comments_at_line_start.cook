-- This is a comment
Mix the ingredients.