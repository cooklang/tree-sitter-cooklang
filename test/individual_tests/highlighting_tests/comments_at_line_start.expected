(recipe
  (comment)
  (step
    (plain_text)))